### ISSUE TITLE
Geth doesn't ask to unlock account for new contract

### ISSUE THREAD

**Issue by hyperh** [ethereum/go-ethereum#1700]
I'm doing the [Greeter tutorial](https://www.ethereum.org/greeter) and at the following part:

````
var greeter = greeterContract.new(_greeting,{from:web3.eth.accounts[0], ...
````

Geth doesn't prompt me to unlock my account. I know it's locked because the transaction doesn't go through and the `console.log()` specified in the greeterContract callback don't show.

How can I get Geth to prompt me for my account passphrase when I make a new contract?

**Comment by maran** [comment]
Just to have some more information, which version are you on?

**Comment by hyperh** [comment]
I was on 1.0.1. I was pasting my code into geth attach. Geth will not prompt for any passphrase, and will not execute a transaction.

Pasting the code into geth console 2>> /dev/null, will work properly. Geth will prompt me for my passphrase before executing a transaction.

**Comment by bas-vk** [comment]
Fixed in #1710
